static/templates/project/base.html

```diff
@@ -11,9 +11,9 @@
       <div class="project-form">
           <h3 class="step">{{project.getStepName(project.getStepId())}}</h3>
           <div class="step-content">
-              <ng-include ng-show="project.getStepId()==1" src="'/static/templates/project/categories.html'"></ng-include>
-              <ng-include ng-show="project.getStepId()==2" src="'/static/templates/project/details.html'"></ng-include>
-              <ng-include ng-show="project.getStepId()==3" src="'/static/templates/project/milestones.html'"></ng-include>
+              <ng-include ng-show="project.getStepId()==1" src="'/static/templates/project/gettingstarted.html'"></ng-include>
```

Comment:
no need to rename files, if we are already under /project we know that the details are of the project